### PR TITLE
Added plugin siocontrol

### PR DESCRIPTION
**Pull request by jcassel**

### CODE REVIEW

File: _plugins/siocontrol.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+- filament
+- usb
+- serial
+- rs232
+- led
+- windows
+
+
+screenshots:
+- url: /assets/img/plugins/SIOControl/SettingsExampleConn.PNG
```
Comment (jneilliii) [inline _plugins/siocontrol.md:45]: URLs are case sensitive, change `SIOControl` to `siocontrol` throughout.

File: _plugins/siocontrol.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+title: SIO Control
+description: Serial IO Control, an alterative way to control locally connected IO. Works on any device(PC,SBC) and OS that supports OctoPrint including Windows.  
+
+authors: 
+- jcassel
+license: AGPLv3
+date: 2023-05-02
+homepage: https://github.com/jcassel/OctoPrint-Siocontrol
+source: https://github.com/jcassel/OctoPrint-Siocontrol
+archive: https://github.com/jcassel/OctoPrint-Siocontrol/archive/master.zip
```
Comment (jneilliii) [inline _plugins/siocontrol.md:13]: should be `main.zip` instead of `master.zip` to match branch naming on GitHub.

**Review by jneilliii — CHANGES_REQUESTED** [review]
along with the comments in this review, you need to modify the returned dictionary [here](https://github.com/jcassel/OctoPrint-Siocontrol/blob/64a24777ee25152a969ed54179617725ec68f636/octoprint_siocontrol/__init__.py#L436) in your software update hook to be `siocontrol` to match the id and plugin_identifier.

File: _plugins/siocontrol.md (hunk 2 above)
Comment (jcassel) [reply _plugins/siocontrol.md]: Corrections made.

**Review by jneilliii — APPROVED** [review]

**Pull request merged**